app/controllers/article_controller.rb

```diff
@@ -39,7 +39,11 @@ def ignore_request_unless_permitted
     return if permitted?
 
     flash_notice(:permission_denied.t)
+    # rubocop disable Style/AndOr
+    # RuboCop 0.83 autocorrects the following line to:
+    #   redirect_to(action: "index_article") && (return)
     redirect_to(action: "index_article") and return
```

Comment:
The attempted autocorrect has redundant parens and, more important, feels wrong -- doesn't read like idiomatic Rails.
But would it work?
```ruby
redirect_to(action: "index_article") && return
```
Note also that when `.rubocop_todo.yml` gets regenerated, it still flags this line as an offense (despite the surrounding disable/enable. 
Is there a way to prevent that?

Reply:
Just delete the `and return` entirely.  It is providing no value in this case.